### ISSUE TITLE
Internationalization

### ISSUE THREAD

**Issue by ZeppLu** [yakGPT/yakGPT#9]
If only i18n is implemented, I would be able to share the site to my family who don't know English.

**Comment by ZainCheung** [comment]
I agree with your suggestion, which will definitely make it easier for more people to talk with ai. It's a awesome project!

**Comment by FlatMapIO** [comment]
I have just added support for typesafe-i18n in the local branch, and I am happy to sync changes upstream if the PR is accepted.

**Comment by yakGPT** [comment]
I would be very happy to accept PRs adding i18n!

**Comment by FlatMapIO** [comment]
typesafe-i18n does not seem to be particularly convenient when combined with the old pages structure of nextjs, I will evaluate the solution and try again.
